src/applications/ezr/containers/IntroductionPage.jsx

```diff
@@ -27,13 +27,16 @@ const IntroductionPage = ({ fetchEnrollmentStatus, route }) => {
 
   const onVerifyEvent = recordEvent({ event: AUTH_EVENTS.VERIFY });
 
-  useEffect(() => {
-    focusElement('.va-nav-breadcrumbs-list');
-    if (isUserLOA3) {
-      fetchEnrollmentStatus();
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
+  useEffect(
+    () => {
+      focusElement('.va-nav-breadcrumbs-list');
+      if (isUserLOA3) {
+        fetchEnrollmentStatus();
+      }
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+    },
+    [isUserLOA3],
```

Comment:
Would we be able to add a test for this in `ezr/tests/unit/containers/IntroductionPage.unit.spec.js`?